_gsocproposals/2020/proposal_TMVAConv3D.md

```diff
@@ -0,0 +1,32 @@
+---
+title: TMVA Deep Learning Developments - 3D Convolutions
+layout: gsoc_proposal
+project: TMVA
+year: 2020
+organization:
+  - CERN
+  - OProject
+---
+
+# Description
+
+Toolkit for Multivariate Analysis (TMVA) is a multi-purpose machine learning toolkit integrated into the ROOT scientific software framework, used in many particle physics data analysis and applications. We have expanded TMVA's capabilities to include a deep learning library (DNN) supporting 2D convolutional layers with training on both CPU and GPUs. This summer we would like to expand the toolkit with 3D convolutional operators implemented both for CPU and GPU. For the GPU the implementation will be based on the cuDNN library 
```

Comment:
"implemented, both for CPU and GPU. For the GPU case, the"

Reply:
Missing a break between library and 3D CNNs.

Reply:
"3D CNNs have very promising applications in particle physics, such as  imaging calorimetry and particle tracking," (comma and doble space)